### ISSUE TITLE
example from fedora-messaging/docs/consuming.rst doesn't work

### ISSUE THREAD

**Issue by praiskup** [fedora-infra/fedora-messaging#186]
I took the consumer script snippet, created `consume.py` from it, added shebang -> and it fails to print the messages.

```
$ ./consume.py
AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
```

So I thought (even if it doesn't seem to) that I have to provide FEDORA_MESSAGING_CONF variable, but it still doesn't work:
```
$ FEDORA_MESSAGING_CONF=/etc/fedora-messaging/fedora.toml ./consume.py
... NO OUTPUT ...
```

While the provided command works with that configuration:
```
$ fedora-messaging --conf /etc/fedora-messaging/fedora.toml  consume
.. I get messages here without problems ...
```

**Comment by jeremycline** [comment]
I assume you mean the snippet in https://fedora-messaging.readthedocs.io/en/stable/consuming.html?

I think this is a duplicate of #171. The command line uses the asynchronous APIs and the asynchronous code is completely separate from the synchronous versions. https://github.com/fedora-infra/fedora-messaging/pull/184 unifies the two and should address #171, but it needs a bit more polish I think before it's really ready.

**Comment by praiskup** [comment]
> I assume you mean the snippet in https://fedora-messaging.readthedocs.io/en/stable/consuming.html?

Yes.

> I think this is a duplicate of #171

If that means that the snippet will start working after #184, yes.

**Comment by jeremycline** [comment]
Okay, then I'm going to close this as a duplicate of #171. #184 makes the snippet referenced work with the Fedora permissions and the /etc/fedora-messaging/fedora.toml config.
